src/main/webapp/app/entities/car/current.tsx

```diff
@@ -34,6 +34,25 @@ const CarCurrentRow = ({ imageUrl, model, status, statusMessage, func, from, to
           </div>
         </div>
       </Col>
+      <Col style={{ textAlign: 'right' }}>
+        <div style={{ height: 150, marginTop: 55 }}>
+          <Button color="primary" onClick={toggle}>
+            {' '}
+            <FontAwesomeIcon icon="home" /> Zwróć wcześniej !
```

Comment:
Tekst na przycisku jest niezgodny z wymaganiami w zadaniu.

Reply:
To było prośba marketingu